src/device-state/target-state.ts

```diff
@@ -0,0 +1,157 @@
+import type { TargetState } from '../types/state';
+import { EventEmitter } from 'events';
+import { InternalInconsistencyError } from '../lib/errors';
+import { getRequestInstance } from '../lib/request';
+import * as url from 'url';
+import * as config from '../config';
+import type { Headers } from 'request';
+import { delay } from 'bluebird';
+import constants = require('../lib/constants');
+import * as _ from 'lodash';
+import { writeLock } from '../lib/update-lock';
+import Bluebird = require('bluebird');
+import type StrictEventEmitter from 'strict-event-emitter-types';
```

Comment:
Can you split the local and external dependencies into groups please?

Reply:
I've added this change.